### ISSUE TITLE
[Error]: Currently installed version of wallaby.js is not covered by your license.

### ISSUE THREAD

**Issue by gadicc** [wallabyjs/public#557]
Title says it all.  Not sure what's going on here, I bought a Freedom license 2 months ago.

Sublime Text v3 - Dev Channel, Build 3111
Linux

The "upgrade and renewal" form says "None of the supplied keys can be upgraded or renewed."

The "latest covered version" form says "Your license has not yet expired, please find your download options below." but only lists IntelliJ and Visual Studio 2013/5?

**Comment by ArtemGovorov** [comment]
What version of Wallaby Sublime package do you have? 
Did you enter your Freedom license key into Wallaby Sublime Package, [as described here](https://wallabyjs.com/docs/intro/license.html#sublime-text)?

**Comment by ArtemGovorov** [comment]
I have just tried your license key in Sublime and it works fine for me. Have also responded to your email, let's try to sort out the issue there.

**Comment by gadicc** [comment]
For anyone else who experienced this, I got super fast help by email from Artem.  The fix was to simply re-enter the license:

* Command Pallette -> Wallaby.js: Show License
* Remove the existing key which didn't match the original I got emailed
* Copy and paste the original key from email

It's unclear how/why it changed, but it was very easy to fix.  I also got tripped up by the fact that Wallaby wasn't showing up in the command palette, I thought because of the license issue, but Wallaby had crashed for a different reason, and as soon as I restarted Sublime, it was trivial to fix this.
